sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -1,30 +1,24 @@
-import time
 import uuid
-from dataclasses import asdict, dataclass
 from datetime import date, datetime, timedelta
-from typing import List, Optional, Set, Union
+from typing import Dict, List, Optional, Union
 
+import numpy as np
 import pandas
 import pyarrow
-from jinja2 import BaseLoader, Environment
-from pandas import Timestamp
 from pydantic import StrictStr
 from pydantic.typing import Literal
 from tenacity import Retrying, retry_if_exception_type, stop_after_delay, wait_fixed
 
-from feast import errors
 from feast.data_source import DataSource
 from feast.errors import (
     BigQueryJobCancelled,
     BigQueryJobStillRunning,
     FeastProviderLoginError,
+    InvalidEntityType,
 )
 from feast.feature_view import FeatureView
+from feast.infra.offline_stores import utils
```

Comment:
Ok this is a mega nitpick, and not a blocking comment: Do you think it would be more descriptive to call `utils` `offline_utils` instead? It would be slightly more verbose, but it changes all the call sites from
```
utils.get_expected_join_keys(project, feature_views, registry)
```
to
```
offline_utils.get_expected_join_keys(project, feature_views, registry)
```
which I feel is a little bit clearer. We are going to have a lot of `utils` over time, and it may not be clear where they are located if we just look at the call site. I also realize that its possible to alias when importing, but I want to avoid having to ask all committers to rename imports every time.

Happy to hear your thoughts.